### PR TITLE
[API]added missing serialization to address editing endpoint

### PR DESCRIPTION
**Pull request by arti0090**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | this task extends #11973
| License         | MIT

### CODE REVIEW

**Review by lchrusciel — APPROVED** [review]

**Review by Tomanhez — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -136,7 +136,7 @@
         "phpspec/phpspec": "^6.1",
         "phpstan/phpstan": "0.12.29",
         "phpstan/phpstan-doctrine": "0.12.16",
-        "phpunit/phpunit": "^8.5",
+        "phpunit/phpunit": "^9.4",
```
Comment (lchrusciel) [inline composer.json:139]: Should be part of separate PR (or mentioned in title)

**Comment by lchrusciel** [comment]
Thanks, @arti0090! :tada:
